mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb

```diff
@@ -254,6 +255,7 @@ func expandNetworkInterfaces(d TerraformResourceData, config *Config) ([]*comput
 			Subnetwork:    sf.RelativeLink(),
 			AccessConfigs: expandAccessConfigs(data["access_config"].([]interface{})),
 			AliasIpRanges: expandAliasIpRanges(data["alias_ip_range"].([]interface{})),
+			NicType:       data["nic_type"].(string),
```

Comment:
Looks like we need different handling of this field when nic_type is not set. I believe it is coming back as nil which cannot be type asserted to a string. You may want to try something like:

Probably want to check for nil and set the value to `""` instead

Reply:
Added. Also added its support for `google_compute_instance_template`